v1.0/deploy-cockroachdb-to-production.md

```diff
@@ -0,0 +1,424 @@
+
+### Topology & Availability Zones
+
+Because CockroachDB doesn't have an intrinsic understanding of your deployment's topology, you must manually identify the node's location (also known as "availability zones") using the `--locality` flag when starting your nodes. CockroachDB then uses these flags as a way of replicating data in a way that ensures survivability.
+
+Your nodes' `--locality` flag should express a hierarchy of topologies. For example:
+
+1. Country
+2. Region
+3. Datacenter
+
+For a deployment in the United States, your `--locality` flags might look like: 
+
+- `--locality=country=us,region=east,datacenter=us-east-1`
+- `--locality=country=us,region=east,datacenter=us-east-2`
+- `--locality=country=us,region=central,datacenter=us-central-1`
+- `--locality=country=us,region=central,datacenter=us-central-2`
+- `--locality=country=us,region=west,datacenter=us-west-1`
+- `--locality=country=us,region=west,datacenter=us-west-2`
+
+{{site.data.alerts.callout_info}}When replicating across datacenters, it's recommended to use datacenters on a single continent to ensure performance (inter-continent scenarios will improve in performance soon).{{site.data.alerts.end}}
+
+For details about controlling the number and location of replicas, see [Configure Replication Zones](configure-replication-zones.html).
+
+## Deployment Methods
+
+Because of CockroachDB's cloud-native design, it works best in conjunction with an **orchestration tool**, such as Kubernetes.
+
+It is certainly possible to run CockroachDB manually, but tasks such as monitoring node liveliness and scaling your cluster are handled much more efficiently with an orchestration tool.
+
+### Load Balancing
+
+If you opt to run CockroachDB outside of an orchestration tool, you will also need to be sure to configure a load balancer manually. This include setting up things like service discovery to ensure that you are aware when new nodes are added to your cluster.
+
+### Service Discovery
+
+If you choose to deploy CockroachDB without an orchestration tool, you'll also need to manually set up a service discovery tool, such as Zookeeper.
+
+At this time, CockroachDB doesn't offer native integration with any service discovery tools, so you will need to manually register nodes with your service discovery tool after starting your node.
+
+### Monitoring
+
+While an orchestration tool can automatically bring up new resources if machines fail, if you choose to deploy CockroachDB manually, you'll need to integrate it with a monitoring tool to alert your team of node failures.
+
+You can monitor your cluster with basically any tool, but we have documentation on integrating CockroachDB with [Prometheus](monitor-cockroachdb-with-prometheus.html).
+
+## Security
+
+To deploy CockroachDB, you *must* use SSL encryption. To make this as simple as possible, the `cockroach` binary can work a certificate authority (CA), which provides everything you need to generate SSLs.
```

Comment:
I think you meant `the cockroach binary can work as a certificate authority`, which isn't really true, we just create certs and keys.
Instead, maybe something like: `the cockroach binary provides everything you need ...`